### PR TITLE
added meta html values

### PR DESCRIPTION
**Pull request by ktsrivastava29**
fix: #3033 

**Problem**
The html_meta values were missing from most of the pages. These values are used to improve SEO. Omitting them results in errors when building the docs with Sphinx.

**Solution offered**
 I've added the meta html values in every page with the exception of three pages, as their documentation is currently being built.

### CODE REVIEW

**Comment by stevepiercy** [comment]
@ktsrivastava29 please add a changelog entry.

**Comment by ktsrivastava29** [comment]
@stevepiercy Sorry, I missed adding that earlier, I've added it now

File: docs/source/blocks/introduction.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
 html_meta:
-  "description": ""
-  "property=og:description": ""
-  "property=og:title": ""
-  "keywords": ""
+  "description": "How to manually enable Blocks on a content type"
+  "property=og:description": "How to manually enable Blocks on a content type"
+  "property=og:title": "Blocks"
+  "keywords": "Volto, Plone, frontend, React"
```
Comment (stevepiercy) [inline docs/source/blocks/introduction.md:6]:
```suggestion
  "keywords": "Volto, Plone, frontend, React, blocks"
```

File: docs/source/blocks/settings.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
 ---
 html_meta:
-  "description": ""
-  "property=og:description": ""
-  "property=og:title": ""
-  "keywords": ""
+  "description": "How to configure a new block"
```
Comment (stevepiercy) [inline docs/source/blocks/settings.md:3]:
```suggestion
  "description": "How to configure custom blocks"
```

File: docs/source/blocks/settings.md [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
+  "property=og:description": "How to configure a new block"
```
Comment (stevepiercy) [inline docs/source/blocks/settings.md:4]:
```suggestion
  "property=og:description": "How to configure custom blocks"
```

File: docs/source/blocks/ssr.md [hunk 4]
```diff
@@ -1,9 +1,9 @@
 ---
 html_meta:
-  "description": ""
-  "property=og:description": ""
-  "property=og:title": ""
-  "keywords": ""
+  "description": "Learn to render async-fetched data that doesn't get rendered fully in the server-side rendering phase"
+  "property=og:description": "Learn to render async-fetched data that doesn't get rendered fully in the server-side rendering phase"
+  "property=og:title": "Server-side rendering for async blocks"
+  "keywords": "Volto, Plone, frontend, React, Render, async-block"
```
Comment (stevepiercy) [inline docs/source/blocks/ssr.md:6]:
```suggestion
  "keywords": "Volto, Plone, frontend, React, Render, async, block"
```

File: docs/source/configuration/backend.md [hunk 5]
```diff
@@ -1,9 +1,9 @@
 ---
 html_meta:
-  "description": ""
-  "property=og:description": ""
-  "property=og:title": ""
-  "keywords": ""
+  "description": "Prepare the Plone backend content API for volto in order to fully support all volto features"
```
Comment (stevepiercy) [inline docs/source/configuration/backend.md:3]:
```suggestion
  "description": "Prepare the Plone backend content API for Volto in order to fully support all Volto features"
```

File: docs/source/configuration/backend.md [hunk 6]
```diff
@@ -1,9 +1,9 @@
...
+  "property=og:description": "Prepare the Plone backend content API for volto in order to fully support all volto features"
```
Comment (stevepiercy) [inline docs/source/configuration/backend.md:4]:
```suggestion
  "property=og:description": "Prepare the Plone backend content API for Volto in order to fully support all Volto features"
```

File: docs/source/configuration/expanders.md [hunk 7]
```diff
@@ -1,9 +1,9 @@
 ---
 html_meta:
-  "description": ""
-  "property=og:description": ""
-  "property=og:title": ""
-  "keywords": ""
+  "description": "Learn to configure the API expanders Volto uses using the `settings.apiExpanders` like this"
```
Comment (stevepiercy) [inline docs/source/configuration/expanders.md:3]:
```suggestion
  "description": "Configure the API expanders in Volto using the `settings.apiExpanders`"
```

File: docs/source/configuration/expanders.md [hunk 8]
```diff
@@ -1,9 +1,9 @@
...
+  "property=og:description": "Learn to configure the API expanders Volto uses using the `settings.apiExpanders` like this"
```
Comment (stevepiercy) [inline docs/source/configuration/expanders.md:4]:
```suggestion
  "property=og:description": "Configure the API expanders in Volto using the `settings.apiExpanders`"
```

File: docs/source/configuration/locking.md [hunk 9]
```diff
@@ -1,9 +1,9 @@
 ---
 html_meta:
-  "description": ""
-  "property=og:description": ""
-  "property=og:title": ""
-  "keywords": ""
+  "description": "Content edit locking feature is to prevent simultaneous conflicting edits of the same content."
```
Comment (stevepiercy) [inline docs/source/configuration/locking.md:3]:
```suggestion
  "description": "Content edit locking prevents simultaneous conflicting edits of the same content."
```

File: docs/source/configuration/locking.md [hunk 10]
```diff
@@ -1,9 +1,9 @@
...
+  "property=og:description": "Content edit locking feature is to prevent simultaneous conflicting edits of the same content."
```
Comment (stevepiercy) [inline docs/source/configuration/locking.md:4]:
```suggestion
  "property=og:description": "Content edit locking prevents simultaneous conflicting edits of the same content."
```

File: docs/source/configuration/richeditor-settings.md [hunk 11]
```diff
@@ -1,9 +1,9 @@
 ---
 html_meta:
-  "description": ""
-  "property=og:description": ""
-  "property=og:title": ""
-  "keywords": ""
+  "description": "In this chapter we will learn how to change settings for the Rich Text Editor."
```
Comment (stevepiercy) [inline docs/source/configuration/richeditor-settings.md:3]:
```suggestion
  "description": "How to change settings for the rich text editor."
```

File: docs/source/configuration/richeditor-settings.md [hunk 12]
```diff
@@ -1,9 +1,9 @@
...
+  "property=og:description": "In this chapter we will learn how to change settings for the Rich Text Editor."
```
Comment (stevepiercy) [inline docs/source/configuration/richeditor-settings.md:4]:
```suggestion
  "property=og:description": "How to change settings for the rich text editor."
```

File: docs/source/contributing/index.md [hunk 13]
```diff
@@ -1,9 +1,9 @@
 ---
 html_meta:
-  "description": ""
-  "property=og:description": ""
-  "property=og:title": ""
-  "keywords": ""
+  "description": "Since the very beginning, Volto has been developed with a set of principles in mind, brought from the years of experience developing Plone"
```
Comment (stevepiercy) [inline docs/source/contributing/index.md:3]:
```suggestion
  "description": "Volto design principles"
```

File: docs/source/contributing/index.md [hunk 14]
```diff
@@ -1,9 +1,9 @@
...
+  "property=og:description": "Since the very beginning, Volto has been developed with a set of principles in mind, brought from the years of experience developing Plone"
```
Comment (stevepiercy) [inline docs/source/contributing/index.md:4]:
```suggestion
  "property=og:description": "Volto design principles"
```

File: docs/source/deploying/index.md [hunk 15]
```diff
@@ -1,9 +1,9 @@
 ---
 html_meta:
-  "description": ""
-  "property=og:description": ""
-  "property=og:title": ""
-  "keywords": ""
+  "description": "A guide to deployment"
```
Comment (stevepiercy) [inline docs/source/deploying/index.md:3]:
```suggestion
  "description": "A guide to Volto frontend deployment"
```

File: docs/source/deploying/index.md [hunk 16]
```diff
@@ -1,9 +1,9 @@
...
+  "property=og:description": "A guide to deployment"
```
Comment (stevepiercy) [inline docs/source/deploying/index.md:4]:
```suggestion
  "property=og:description": "A guide to Volto frontend deployment"
```

File: docs/source/deploying/simple.md [hunk 17]
```diff
@@ -1,9 +1,9 @@
 ---
 html_meta:
-  "description": ""
-  "property=og:description": ""
-  "property=og:title": ""
-  "keywords": ""
+  "description": "Volto is a Node application that runs on your machine/server and listens to a port. Once you are ready to deploy it, you should build it running:"
```
Comment (stevepiercy) [inline docs/source/deploying/simple.md:3]:
```suggestion
  "description": "Simple deployment of a Volto application"
```

File: docs/source/deploying/simple.md [hunk 18]
```diff
@@ -1,9 +1,9 @@
...
+  "property=og:description": "Volto is a Node application that runs on your machine/server and listens to a port. Once you are ready to deploy it, you should build it running:"
```
Comment (stevepiercy) [inline docs/source/deploying/simple.md:4]:
```suggestion
  "property=og:description": "Simple deployment of a Volto application"
```

File: docs/source/developer-guidelines/acceptance-tests.md [hunk 19]
```diff
@@ -1,9 +1,9 @@
 ---
 html_meta:
-  "description": ""
-  "property=og:description": ""
-  "property=og:title": ""
-  "keywords": ""
+  "description": "Developer guidelines for acceptance tests."
+  "property=og:description": "Developer guidelines for acceptance tests."
+  "property=og:title": "Acceptance tests"
+  "keywords": "Volto, Plone, frontend, React, helper command, redux"
```
Comment (stevepiercy) [inline docs/source/developer-guidelines/acceptance-tests.md:6]:
```suggestion
  "keywords": "Volto, Plone, frontend, React, helper command, redux, acceptance, tests, Cypress"
```

File: docs/source/developer-guidelines/design-principles.md [hunk 20]
```diff
@@ -1,9 +1,13 @@
 ---
 html_meta:
-  "description": ""
-  "property=og:description": ""
-  "property=og:title": ""
-  "keywords": ""
+  "description": "Volto has been developed with a set of principles in mind,
```
Comment (stevepiercy) [inline docs/source/developer-guidelines/design-principles.md:3]:
```suggestion
  "description": "Volto design principles"
```

File: docs/source/developer-guidelines/design-principles.md [hunk 21]
```diff
@@ -1,9 +1,13 @@
...
+brought from the years of experience developing Plone core and implementing projects on
```
Comment (stevepiercy) [inline docs/source/developer-guidelines/design-principles.md:4]:
```suggestion
```

File: docs/source/developer-guidelines/design-principles.md [hunk 22]
```diff
@@ -1,9 +1,13 @@
...
+it."
```
Comment (stevepiercy) [inline docs/source/developer-guidelines/design-principles.md:5]:
```suggestion
```

File: docs/source/developer-guidelines/design-principles.md [hunk 23]
```diff
@@ -1,9 +1,13 @@
...
+  "property=og:description": "Volto has been developed with a set of principles in mind,
```
Comment (stevepiercy) [inline docs/source/developer-guidelines/design-principles.md:6]:
```suggestion
  "property=og:description": "Volto design principles"
```

File: docs/source/developer-guidelines/design-principles.md [hunk 24]
```diff
@@ -1,9 +1,13 @@
...
+brought from the years of experience developing Plone core and implementing projects on
```
Comment (stevepiercy) [inline docs/source/developer-guidelines/design-principles.md:7]:
```suggestion
```

File: docs/source/developer-guidelines/design-principles.md [hunk 25]
```diff
@@ -1,9 +1,13 @@
...
+it."
```
Comment (stevepiercy) [inline docs/source/developer-guidelines/design-principles.md:8]:
```suggestion
```

File: docs/source/developer-guidelines/linting.md [hunk 26]
```diff
@@ -1,9 +1,11 @@
 ---
 html_meta:
-  "description": ""
-  "property=og:description": ""
-  "property=og:title": ""
-  "keywords": ""
+  "description": "Volto developers can enjoy a lot of freedom in their choice of text editors and
```
Comment (stevepiercy) [inline docs/source/developer-guidelines/linting.md:3]:
```suggestion
  "description": "Code linting in Volto"
```

File: docs/source/developer-guidelines/linting.md [hunk 27]
```diff
@@ -1,9 +1,11 @@
...
+IDEs"
```
Comment (stevepiercy) [inline docs/source/developer-guidelines/linting.md:4]:
```suggestion
```

File: docs/source/developer-guidelines/linting.md [hunk 28]
```diff
@@ -1,9 +1,11 @@
...
+  "property=og:description": "Volto developers can enjoy a lot of freedom in their choice of text editors and
```
Comment (stevepiercy) [inline docs/source/developer-guidelines/linting.md:5]:
```suggestion
  "property=og:description": "Code linting in Volto"
```

File: docs/source/developer-guidelines/linting.md [hunk 29]
```diff
@@ -1,9 +1,11 @@
...
+IDEs"
```
Comment (stevepiercy) [inline docs/source/developer-guidelines/linting.md:6]:
```suggestion
```

File: docs/source/getting-started/roadmap.md [hunk 30]
```diff
@@ -1,9 +1,9 @@
 ---
 html_meta:
-  "description": ""
-  "property=og:description": ""
-  "property=og:title": ""
-  "keywords": ""
+  "description": "A roadmap to learn Volto."
```
Comment (stevepiercy) [inline docs/source/getting-started/roadmap.md:3]:
```suggestion
  "description": "Volto developer roadmap"
```

File: docs/source/getting-started/roadmap.md [hunk 31]
```diff
@@ -1,9 +1,9 @@
...
+  "property=og:description": "A roadmap to learn Volto."
```
Comment (stevepiercy) [inline docs/source/getting-started/roadmap.md:4]:
```suggestion
  "property=og:description": "Volto developer roadmap"
```

File: docs/source/recipes/appextras.md [hunk 32]
```diff
@@ -1,9 +1,10 @@
 ---
 html_meta:
-  "description": ""
-  "property=og:description": ""
-  "property=og:title": ""
-  "keywords": ""
+  "description": "The `AppExtras` component is a general use insertion point for things like
```
Comment (stevepiercy) [inline docs/source/recipes/appextras.md:3]:
```suggestion
  "description": "The `AppExtras` component is a general use insertion point for general purpose code spanning the whole application or for third party services code."
```

File: docs/source/recipes/appextras.md [hunk 33]
```diff
@@ -1,9 +1,10 @@
...
+analytics, general purpose code spanning the whole application or third party services code."
```
Comment (stevepiercy) [inline docs/source/recipes/appextras.md:4]:
```suggestion
```

File: docs/source/recipes/appextras.md [hunk 34]
```diff
@@ -1,9 +1,10 @@
...
+  "property=og:description": "The `AppExtras` component is a general use insertion point for things like analytics, general purpose code spanning the whole application or third party services code."
```
Comment (stevepiercy) [inline docs/source/recipes/appextras.md:5]:
```suggestion
  "property=og:description": "The `AppExtras` component is a general use insertion point for general purpose code spanning the whole application or for third party services code."
```

File: docs/source/recipes/creating-project.md [hunk 35]
```diff
@@ -1,9 +1,9 @@
 ---
 html_meta:
-  "description": ""
-  "property=og:description": ""
-  "property=og:title": ""
-  "keywords": ""
+  "description": "Gain an understanding of how to create a new Volto project."
```
Comment (stevepiercy) [inline docs/source/recipes/creating-project.md:3]:
```suggestion
  "description": "Creating a new Volto project"
```

File: docs/source/recipes/creating-project.md [hunk 36]
```diff
@@ -1,9 +1,9 @@
...
+  "property=og:description": "Gain an understanding of how to create a new Volto project."
```
Comment (stevepiercy) [inline docs/source/recipes/creating-project.md:4]:
```suggestion
  "property=og:description": "Creating a new Volto project"
```

File: docs/source/recipes/creating-views.md [hunk 37]
```diff
@@ -1,9 +1,10 @@
 ---
 html_meta:
-  "description": ""
-  "property=og:description": ""
-  "property=og:title": ""
-  "keywords": ""
+  "description": "In this chapter we are going to create a new type of view for displaying
```
Comment (stevepiercy) [inline docs/source/recipes/creating-views.md:3]:
```suggestion
  "description": "Creating Volto views"
```

File: docs/source/recipes/creating-views.md [hunk 38]
```diff
@@ -1,9 +1,10 @@
...
+contents in a folder. We will call this view `full view`."
```
Comment (stevepiercy) [inline docs/source/recipes/creating-views.md:4]:
```suggestion
```

File: docs/source/recipes/creating-views.md [hunk 39]
```diff
@@ -1,9 +1,10 @@
...
+  "property=og:description": "In this chapter we are going to create a new type of view for displaying contents in a folder. We will call this view `full view`."
```
Comment (stevepiercy) [inline docs/source/recipes/creating-views.md:5]:
```suggestion
  "property=og:description": "Creating Volto views"
```

File: docs/source/recipes/customizing-views.md [hunk 40]
```diff
@@ -1,9 +1,9 @@
 ---
 html_meta:
-  "description": ""
-  "property=og:description": ""
-  "property=og:title": ""
-  "keywords": ""
+  "description": "Learn how to customize volto views."
```
Comment (stevepiercy) [inline docs/source/recipes/customizing-views.md:3]:
```suggestion
  "description": "Customize Volto views."
```

File: docs/source/recipes/customizing-views.md [hunk 41]
```diff
@@ -1,9 +1,9 @@
...
+  "property=og:description": "Learn how to customize volto views."
```
Comment (stevepiercy) [inline docs/source/recipes/customizing-views.md:4]:
```suggestion
  "property=og:description": "Customize Volto views."
```

File: docs/source/recipes/express.md [hunk 42]
```diff
@@ -1,9 +1,9 @@
 ---
 html_meta:
-  "description": ""
-  "property=og:description": ""
-  "property=og:title": ""
-  "keywords": ""
+  "description": "Volto uses the popular Express server for its Server-Side Rendering implementation and static resource serving."
+  "property=og:description": "Volto uses the popular Express server for its Server-Side Rendering implementation and static resource serving."
+  "property=og:title": "Custom Express middleware"
+  "keywords": "Volto, Plone, frontend, React, middleware"
```
Comment (stevepiercy) [inline docs/source/recipes/express.md:6]:
```suggestion
  "keywords": "Volto, Plone, frontend, React, custom, express, middleware"
```

File: docs/source/recipes/express.md [hunk 43]
```diff
@@ -1,9 +1,9 @@
 ---
 html_meta:
-  "description": ""
-  "property=og:description": ""
-  "property=og:title": ""
-  "keywords": ""
+  "description": "Volto uses the popular Express server for its Server-Side Rendering implementation and static resource serving."
```
Comment (stevepiercy) [inline docs/source/recipes/express.md:3]:
```suggestion
  "description": "Volto uses the popular Express server for its server-side rendering implementation and static resource serving."
```

File: docs/source/recipes/express.md [hunk 44]
```diff
@@ -1,9 +1,9 @@
...
+  "property=og:description": "Volto uses the popular Express server for its Server-Side Rendering implementation and static resource serving."
```
Comment (stevepiercy) [inline docs/source/recipes/express.md:4]:
```suggestion
  "property=og:description": "Volto uses the popular Express server for its server-side rendering implementation and static resource serving."
```

File: docs/source/recipes/folder-structure.md [hunk 45]
```diff
@@ -1,9 +1,11 @@
 ---
 html_meta:
-  "description": ""
-  "property=og:description": ""
-  "property=og:title": ""
-  "keywords": ""
+  "description": "Volto is based on React, Redux, and React-Router. All of the
```
Comment (stevepiercy) [inline docs/source/recipes/folder-structure.md:3]:
```suggestion
  "description": "Volto is based on React, Redux, and React-Router and follows its convention of resource location."
```

File: docs/source/recipes/folder-structure.md [hunk 46]
```diff
@@ -1,9 +1,11 @@
...
+code is located in the `src` folder."
```
Comment (stevepiercy) [inline docs/source/recipes/folder-structure.md:4]:
```suggestion
```

File: docs/source/recipes/folder-structure.md [hunk 47]
```diff
@@ -1,9 +1,11 @@
...
+  "property=og:description": "Volto is based on React, Redux, and React-Router. All of the
```
Comment (stevepiercy) [inline docs/source/recipes/folder-structure.md:5]:
```suggestion
  "property=og:description": "Volto is based on React, Redux, and React-Router and follows its convention of resource location."
```

File: docs/source/recipes/folder-structure.md [hunk 48]
```diff
@@ -1,9 +1,11 @@
...
+code is located in the `src` folder."
```
Comment (stevepiercy) [inline docs/source/recipes/folder-structure.md:6]:
```suggestion
```

File: docs/source/recipes/ie11compat.md [hunk 49]
```diff
@@ -1,9 +1,9 @@
 ---
 html_meta:
-  "description": ""
-  "property=og:description": ""
-  "property=og:title": ""
-  "keywords": ""
+  "description": "Legacy Browser Support (IE11 compatibility): Polyfills, babel-env, Pre-transpiling"
+  "property=og:description": "Legacy Browser Support (IE11 compatibility): Polyfills, babel-env, Pre-transpiling"
+  "property=og:title": "Legacy Browser Support"
+  "keywords": "Volto, Plone, frontend, React, legacy browser support"
```
Comment (stevepiercy) [inline docs/source/recipes/ie11compat.md:6]:
```suggestion
  "keywords": "Volto, Plone, frontend, React, legacy browser support, IE11 compatability, polyfills"
```

File: docs/source/recipes/ie11compat.md [hunk 50]
```diff
@@ -1,9 +1,9 @@
 ---
 html_meta:
-  "description": ""
-  "property=og:description": ""
-  "property=og:title": ""
-  "keywords": ""
+  "description": "Legacy Browser Support (IE11 compatibility): Polyfills, babel-env, Pre-transpiling"
```
Comment (stevepiercy) [inline docs/source/recipes/ie11compat.md:3]:
```suggestion
  "description": "Legacy browser support with polyfills, babel-env, and pre- and post-transpiling"
```

File: docs/source/recipes/ie11compat.md [hunk 51]
```diff
@@ -1,9 +1,9 @@
...
+  "property=og:description": "Legacy Browser Support (IE11 compatibility): Polyfills, babel-env, Pre-transpiling"
```
Comment (stevepiercy) [inline docs/source/recipes/ie11compat.md:4]:
```suggestion
  "property=og:description": "Legacy browser support with polyfills, babel-env, and pre- and post-transpiling"
```

File: docs/source/recipes/ie11compat.md [hunk 52]
```diff
@@ -1,9 +1,9 @@
...
+  "property=og:title": "Legacy Browser Support"
```
Comment (stevepiercy) [inline docs/source/recipes/ie11compat.md:5]:
```suggestion
  "property=og:title": "Legacy Browser Support (IE11 compatibility)"
```

File: docs/source/recipes/index.md [hunk 53]
```diff
@@ -1,9 +1,9 @@
 ---
 html_meta:
-  "description": ""
-  "property=og:description": ""
-  "property=og:title": ""
-  "keywords": ""
+  "description": "Recipes: Development"
```
Comment (stevepiercy) [inline docs/source/recipes/index.md:3]:
```suggestion
  "description": "Volto development recipes"
```

File: docs/source/recipes/index.md [hunk 54]
```diff
@@ -1,9 +1,9 @@
...
+  "property=og:description": "Recipes: Development"
```
Comment (stevepiercy) [inline docs/source/recipes/index.md:4]:
```suggestion
  "property=og:description": "Volto development recipes"
```

File: docs/source/recipes/lazyload.md [hunk 55]
```diff
@@ -1,9 +1,9 @@
 ---
 html_meta:
-  "description": ""
-  "property=og:description": ""
-  "property=og:title": ""
-  "keywords": ""
+  "description": "A guide to lazy-loading libraries as these libraries are not as straight-forward as with the React components."
```
Comment (stevepiercy) [inline docs/source/recipes/lazyload.md:3]:
```suggestion
  "description": "A guide to lazy-loading libraries in Volto."
```

File: docs/source/recipes/lazyload.md [hunk 56]
```diff
@@ -1,9 +1,9 @@
...
+  "property=og:description": "A guide to lazy-loading libraries as these libraries are not as straight-forward as with the React components."
```
Comment (stevepiercy) [inline docs/source/recipes/lazyload.md:4]:
```suggestion
  "property=og:description": "A guide to lazy-loading libraries in Volto."
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]

**Comment by ktsrivastava29** [comment]
@stevepiercy Thank you for bringing up the required edits; I've now made the changes you suggested.

File: docs/source/blocks/settings.md [hunk 57]
```diff
@@ -1,7 +1,7 @@
 ---
 html_meta:
-  "description": "How to configure a new block"
-  "property=og:description": "How to configure a new block"
+  "description": "How to configure custom block"
```
Comment (stevepiercy) [inline docs/source/blocks/settings.md:3]:
```suggestion
  "description": "How to configure custom blocks"
```

File: docs/source/blocks/settings.md [hunk 58]
```diff
@@ -1,7 +1,7 @@
...
+  "property=og:description": "How to configure custom block"
```
Comment (stevepiercy) [inline docs/source/blocks/settings.md:4]:
```suggestion
  "property=og:description": "How to configure custom blocks"
```

File: docs/source/recipes/folder-structure.md [hunk 59]
```diff
@@ -1,9 +1,7 @@
 ---
 html_meta:
-  "description": "Volto is based on React, Redux, and React-Router. All of the
-code is located in the `src` folder."
-  "property=og:description": "Volto is based on React, Redux, and React-Router. All of the
-code is located in the `src` folder."
+  "description": ""Volto is based on React, Redux, and React-Router and follows its convention of resource location."
```
Comment (stevepiercy) [inline docs/source/recipes/folder-structure.md:3]:
```suggestion
  "description": "Volto is based on React, Redux, and React-Router and follows its convention of resource location."
```

File: docs/source/recipes/folder-structure.md [hunk 60]
```diff
@@ -1,9 +1,7 @@
...
+  "property=og:description": ""Volto is based on React, Redux, and React-Router and follows its convention of resource location."
```
Comment (stevepiercy) [inline docs/source/recipes/folder-structure.md:4]:
```suggestion
  "property=og:description": "Volto is based on React, Redux, and React-Router and follows its convention of resource location."
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
One more round of corrections.

Do you know that you can simply click a button in GitHub to accept the suggestion? That avoids unnecessary manual fixes.

**Comment by ktsrivastava29** [comment]
@stevepiercy  Yes, actually since there were so many changes, I felt that I should do it locally first. 
Now I've incorporated the changes using the way you advised.

**Review by stevepiercy — APPROVED** [review]

**Comment by stevepiercy** [comment]
Thank you!

**Pull request merged**